modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -31,6 +31,7 @@ The following Technology Preview features are enabled by this feature set:
 ** Dynamic Resource Allocation API. Enables a new API for requesting and sharing resources between pods and containers. This is an internal feature that most users do not need to interact with. (`DynamicResourceAllocation`)
 ** Pod security admission enforcement. Enables the restricted enforcement mode for pod security admission. Instead of only logging a warning, pods are rejected if they violate pod security standards. (`OpenShiftPodSecurityAdmission`)
 ** StatefulSet pod availability upgrading limits. Enables users to define the maximum number of statefulset pods unavailable during updates which reduces application downtime. (`MaxUnavailableStatefulSet`)
+** Admin Network Policy and Baseline Admin Network Policy. Enables `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic. (`AdminNetworkPolicy`)
```

Comment:
@ahardin-rh Should it be "resources, which are part of the Network Policy V2 API" referring to calling out  **V2 API** here?

Reply:
@asood-rh I'm sorry. I don't understand your comment. Can you please clarify? Thanks!

Reply:
I think the sentence is ok, this is just a snippet to let users know what's the new thing about. 
"resources, which are part of the Network Policy V2 API," is what is written right?

Reply:
correct; thanks!

Reply:
@asood-rh 's confusion was around the whole "V2" part which makes sense - reading https://network-policy-api.sigs.k8s.io/ will help.

- NetworkPolicy in tree object is referred to as V1
- ANP, BANP (future DNP) are referred to as V2 of Network Policy APIs fondly.